tensorflow/core/BUILD

```diff
@@ -149,6 +149,7 @@ load(
     "//third_party/mkl:build_defs.bzl",
     "if_mkl",
 )
+load("@io_bazel_rules_closure//closure:defs.bzl","closure_proto_library")
```

Comment:
[nit] space after comma so we don't get unrelated buildifier edits later.

Reply:
done